src/collections/_documentation/error-reporting/getting-started-install/php.md

```diff
@@ -0,0 +1,26 @@
+To install the SDK you will need to be using `Composer` in your project. To install
+it please see the [docs](https://getcomposer.org/download/).
+
+Sentry PHP is not tied to any specific library that sends HTTP messages. Instead,
+it uses `Httplug` to let users choose whichever PSR-7 implementation and HTTP client
+they want to use.
+
+If you just want to get started quickly you should run the following command:
+
+```bash
+php composer.phar require sentry/sentry php-http/curl-client guzzlehttp/psr7
+```
+
+This will install the library itself along with an HTTP client adapter that uses
+cURL as transport method (provided by `Httplug`) and a PSR-7 implementation
+(provided by Guzzle). You do not have to use those packages if you do not want to.
+The SDK does not care about which transport method you want to use because it's
+an implementation detail of your application. You may use any package that provides
+`php-http/async-client-implementation` and `http-message-implementation`.
```

Comment:
We should link to https://packagist.org/providers/php-http/async-client-implementation and https://packagist.org/providers/psr/http-message-implementation